src/controllers/profile.controller.ts

```diff
@@ -0,0 +1,68 @@
+import {repository} from '@loopback/repository';
+import {post, requestBody} from '@loopback/rest';
+import {PogProfileRepository} from '../repositories';
+
+export class ProfileController {
+  constructor(
+    @repository(PogProfileRepository)
+    private pogProfileRepository: PogProfileRepository,
+  ) {}
+
+  @post('/merge')
```

Comment:
merge doesn't convey the right intent as it's one of the consequence.

`consolidate` maybe ?